test/source/tests/settings.ts

```diff
@@ -519,6 +520,46 @@ export let defineSettingsTests = (testVariant: TestVariant, testWithBrowser: Tes
       await settingsPage.close();
     }));
 
+    ava.default('settings - manual backup several keys', testWithBrowser(undefined, async (t, browser) => {
+      const acctEmail = 'flowcrypt.test.key.multiple@gmail.com';
+      const settingsPage1 = await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);
+      await SetupPageRecipe.manualEnter(settingsPage1, 'unused', {
+        submitPubkey: false,
+        usedPgpBefore: false,
+        key: {
+          title: '?',
+          armored: testConstants.testKeyMultiple1b383d0334e38b28,
+          passphrase: '1234',
+          longid: '1b383d0334e38b28',
+        }
+      }, { isSavePassphraseChecked: false, isSavePassphraseHidden: false });
+      await settingsPage1.close();
+      await SettingsPageRecipe.addKeyTest(t, browser, acctEmail, testConstants.testKeyMultiple98acfa1eadab5b92, '1234',
+        { isSavePassphraseChecked: true, isSavePassphraseHidden: false });
+      await SettingsPageRecipe.addKeyTest(t, browser, acctEmail, testConstants.testKeyMultipleSmimeCEA2D53BB9D24871, '1234',
+        { isSavePassphraseChecked: true, isSavePassphraseHidden: false });
+      await SettingsPageRecipe.addKeyTest(t, browser, acctEmail, testConstants.testKeyMultipleSmimeA35068FD4E037879, '1234',
+        { isSavePassphraseChecked: true, isSavePassphraseHidden: false });
+      const backupPage = await browser.newPage(t, TestUrls.extension(`/chrome/settings/modules/backup.htm?acctEmail=${acctEmail}&action=backup_manual&parentTabId=0`));
+      // play with backup controls here
+      expect(await backupPage.isDisabled('[data-fingerprints="47FB03183E03A8ED44E3BBFCCEA2D53BB9D24871"]')).to.equal(true);
+      expect(await backupPage.isDisabled('[data-fingerprints="5A08466253C956E9C76C2E95A35068FD4E037879"]')).to.equal(true);
+      await backupPage.waitAndClick('[data-fingerprints="CB0485FE44FC22FF09AF0DB31B383D0334E38B28,B38FC17737BB1D66C69CB25975796017BA7823CF"]'); // uncheck
```

Comment:
Please also test that these two are initially checked

Reply:
I've added a test to detect if these two checkboxes are "unchecked".